components/body/desktop.styl

```diff
@@ -44,24 +38,19 @@
   .body-section-content
     border-top-style none
 
-  .columns
-  .section1-text
-  .section-three-text-container
-  .section-five-content__part2
+  #genome img
+  .left-gutter-formatter
     padding-left 35px
     width 50%
-  .section-two-text
-  .carousel-track__image
-  .section-three img
-  .section-five-content__part1
-  .flex-right-formatting
+
+  #collecting img
+  .right-gutter-formatter
     padding-right 35px
     width 50%
 
-  .section-three img
-    padding-top 0px
-
-  .columns, .section-three .body-section-content
+  .body-section-content.remove-bottom-margin
+  .body-section-content img
+  .columns
```

Comment:
There's no real need to try to save a few bytes by combining all these declarations. Should be optimizing for modularity and readability.